find_email_addresses.py

```diff
@@ -0,0 +1,45 @@
+import sys, re
+import link_extractor as le
+from collections import deque
+
+debug = False
+
+try:
+	domain = sys.argv[1]
+except:
+	domain = 'web.mit.edu'
+	debug = False
```

Comment:
Was used in testing. Debug was set to true in this default case during some of the development. Either the variable should be removed or better, a debug tag should be allowed for the user to see the program run more verbose-ly